ts/components/dialogs/connect_wallet_dialog.tsx

```diff
@@ -220,14 +220,18 @@ export const ConnectWalletDialog = () => {
         setActivatingConnector(currentConnector);
         onCloseDialog();
         const provider = await currentConnector.getProvider();
+        const providerName = utils.getProviderName(provider);
+
         if (option.type === 'walletconnect') {
             address = provider.accounts[0];
         } else {
             address = provider._addresses ? provider._addresses[0] : provider.selectedAddress;
         }
         if (typeof window !== undefined && address) {
             const connectedData = JSON.stringify({
-                name: option.type,
+                name: providerName.includes('Provider')
+                    ? providerName.replace('Provider', '').toUpperCase()
+                    : providerName.toUpperCase(),
```

Comment:
Can we make a utility function that re-maps the name correctly so we don't have to rename everything?

Reply:
I'm removing this. Its not necessary.